pkg/client/dtclient/config_client.go

```diff
@@ -135,6 +135,14 @@ func (d *ConfigClient) UpsertByName(ctx context.Context, a api.API, name string,
 	if a.ID == api.Extension {
```

Comment:
Also i would do the change in same style you did with slo in `pkg/deploy/internal/setting/setting.go`